src/TinyMCEInput.js

```diff
@@ -82,7 +82,8 @@ var TinyMCEInput = React.createClass({
       pollInterval: 1000,
       textareaProps: {},
       otherEventHandlers: {},
-      onChange: function() {}
+      onChange: function() {},
+      component: 'textinput',
```

Comment:
Was the change from "textarea" to "textinput" intentional?  I'm not sure that is valid, seeing how "textinput" is not a valid HTML tag.